content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,203 @@
+  suitable for handling non-HTTP workloads.
+
+## Gateway API
+
+To overcome this, Gateway API is designed to provide a more flexible, extensible, and powerful way
+to manage traffic to your services.
+
+Gateway API is just weeks away from a GA (General Availability) release. It provides a standard
+Kubernetes API for ingress traffic control. It offers extended functionality, improved
+customization, and greater flexibility. By focusing on modular and expressive API resources, Gateway
+API makes it possible to describe a wider array of routing configurations and models.
+
+The transition from Ingress API to Gateway API in Kubernetes is driven by advantages and advanced
+functionalities that Gateway API offers, with its foundation built on four core principles: a
+role-oriented approach, portability, expressiveness and extensibility.
+
+### A role-oriented approach
+
+Gateway API employs a role-oriented approach that aligns with the conventional roles within
+organizations involved in configuring Kubernetes service networking. This approach enables
+infrastructure engineers, cluster operators, and application developers to collectively address
+different aspects of Gateway API.
+
+For instance, infrastructure engineers play a pivotal role in deploying GatewayClasses,
+cluster-scoped resources that act as templates to explicitly define behavior for Gateways derived
+from them, laying the groundwork for robust service networking.
+
+Subsequently, cluster operators utilize these GatewayClasses to deploy gateways. A Gateway in
+Kubernetes' Gateway API defines how external traffic can be directed to Services within the cluster,
+essentially bridging non-Kubernetes sources to Kubernetes-aware destinations. It represents a
+request for a load balancer configuration aligned with a GatewayClass’ specification. The Gateway
+spec may not be exhaustive as some details can be supplied by the GatewayClass controller, ensuring
+portability. Additionally, a Gateway can be linked to multiple Route references to channel specific
+traffic subsets to designated services.
+
+Lastly, application developers configure route resources (such as HTTPRoutes), to manage
+configuration (e.g. timeouts, request matching/filter) and Service composition (e.g. path routing to
+backends) Route resources define protocol-specific rules for mapping requests from a Gateway to
+Kubernetes Services. HTTPRoute is for multiplexing HTTP or terminated HTTPS connections. It's
+intended for use in cases where you want to inspect the HTTP stream and use HTTP request data for
+either routing or modification, for example using HTTP Headers for routing, or modifying them
+in-flight.
+
+{{< figure src="gateway-api-resources.svg" alt="Diagram showing the APIs that make up Gateway and how they relate to each other. The APIs shown are GatewayClass, Gateway, and HTTPRoute; the Service API is also shown" caption="The APIs that make up Gateway" class="diagram-medium" >}}
```

Comment:
```suggestion
{{< figure src="gateway-api-resources.svg" alt="Diagram showing the key resources that make up Gateway API and how they relate to each other. The resources shown are GatewayClass, Gateway, and HTTPRoute; the Service API is also shown" caption="The resources that make up Gateway API" class="diagram-medium" >}}
```
@sftim For some reason, the caption is not rendered any differently than the text in the [deploy preview](https://deploy-preview-43204--kubernetes-io-main-staging.netlify.app/blog/2023/10/25/introducing-ingress2gateway/#a-role-oriented-approach). Should we be doing something differently?
<img width="686" alt="Screenshot 2023-10-23 at 9 33 39 AM" src="https://github.com/kubernetes/website/assets/419322/726937ba-38eb-481c-8599-cc3610018f6f">

Reply:
I did not commit the new SVG, was waiting for consensus.
If we are all on board with the new one I will go ahead and do it. I saw one comment from you about preferring png

Reply:
🤦 I can be ok with SVG, but apparently I uploaded the wrong screenshot. I was trying to show that the caption looked out of place in the deploy preview:
<img width="594" alt="Screenshot 2023-10-24 at 9 35 08 AM" src="https://github.com/kubernetes/website/assets/419322/f108106f-068e-47a7-8ad8-2b9b698bbd5d">

Reply:
Easy answer, then: omit the caption.